CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 
 - Added `com.unity.test-framework` and `com.unity.test-framework.performance` as dependencies to the `io.improbable.gdk.testutils` package. [#1416](https://github.com/spatialos/gdk-for-unity/pull/1416)
     - In addition, both packages have been removed from the playground `manifest.json`.
+- Updated buildkite artifact paths to only use `/` rather than `\\` as path-seperators. [#1418](https://github.com/spatialos/gdk-for-unity/pull/1418)
```

Comment:
You don't need to add changelog entries for CI-related things

The changelog is more for changes to the GDK itself

Reply:
same comment applies to the FPS/Blank project PRs too

Reply:
Okay, thanks! I'll remove the line